filter/foomatic-rip/pdf.c

```diff
@@ -47,7 +47,7 @@ int pdf_count_pages(const char *filename)
     size_t bytes;
     char *p;
 
-    snprintf(gscommand, CMDLINE_MAX, "%s -dNODISPLAY -q -c "
+    snprintf(gscommand, CMDLINE_MAX, "%s -dNODISPLAY -dSAFER -dNOPAUSE -q -c "
```

Comment:
it seems to me that with `-dSAFER`, you can't open an external file:

```
$ gs -dNODISPLAY -dSAFER -dNOPAUSE -q -c '/pdffile (foomatic-i89GMn) (r) file runpdfbegin (PageCount: ) print pdfpagecount = quit'
Error: /invalidfileaccess in --file--
Operand stack:
   pdffile   (foomatic-i89GMn)   (r)
Execution stack:
   %interp_exit   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--   --nostringval--   --nostringval--   false   1   %stopped_push   .runexec2   --nostringval--   --nostringval--   --nostringval--   2   %stopped_push   --nostringval--
Dictionary stack:
   --dict:1161/1684(ro)(G)--   --dict:0/20(G)--   --dict:77/200(L)--
Current allocation mode is local
Last OS error: 2
GPL Ghostscript 9.07: Unrecoverable error, exit code 1
```

should `-dSAFE` be used instead of ?